code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -141,14 +141,6 @@
 	SEND_SIGNAL(src, COMSIG_TURF_EXPOSE, air, exposed_temperature)
 	check_atmos_process(src, air, exposed_temperature) //Manually do this to avoid needing to use elements, don't want 200 second atom init times
 
-/turf/proc/archive()
-	temperature_archived = temperature
-
-/turf/open/archive()
-	air.archive()
-	archived_cycle = SSair.times_fired
-	temperature_archived = temperature
```

Comment:
Rather then putting gas_mixture on /turf, keep archive, and unroll it just for the stuff in process_cell, using the define in turf/open/archive()